ui/packages/app/src/components/Column/Column.svelte

```diff
@@ -12,7 +12,7 @@
 
 <div
 	id={elem_id}
-	class="overflow-hidden flex flex-col relative col {create_classes(style)}"
+	class="flex flex-col relative col {create_classes(style)}"
```

Comment:
I can't think of any harm that's done by removing `overflow-hidden` here, but can you confirm @pngwn?

Reply:
This is done to allow the dropdown to extend past the parent.

Reply:
We just need to make sure that this wasn't responsible for enforcing rounded borders (which I don't think it is/was, Block does that). Otherwise safe to remove.